AlphaWallet/Tokens/ViewControllers/TokenInstanceViewController.swift

```diff
@@ -121,10 +122,16 @@ class TokenInstanceViewController: UIViewController, TokenVerifiableStatusViewCo
 
         header.configure(viewModel: .init(tokenObject: tokenObject, server: tokenObject.server, assetDefinitionStore: assetDefinitionStore))
 
-        let actions = viewModel.actions
-        buttonsBar.numberOfButtons = actions.count
-        buttonsBar.configure()
-        for (action, button) in zip(actions, buttonsBar.buttons) {
+        buttonsBar.configure(.combined(buttons: viewModel.actions.count))
+        buttonsBar.delegate = self
+        buttonsBar.dataSource = self
+
+        var actions = viewModel.actions
+        actions.removeFirst(buttonsBar.buttons.count)
+        moreActions = actions
+
+        buttonsBar.buttons.enumerated().forEach { (index, button) in
```

Comment:
- [ ] Can you help replace this with `for-each`? I think you changed it the last time. I might have missed it when I merged during the squash operation.